src/views/home/home.view.tsx

```diff
@@ -0,0 +1,470 @@
+}: HomeProps): JSX.Element {
+  const theme = useTheme<Theme>();
+  const classes = useHomeStyles();
+  const ethereumNetworkAndWalletLoaded = isAsyncTaskDataAvailable(ethereumNetworkTask) && wallet;
+  const accountPendingDeposits = React.useMemo(
+    () =>
+      ethereumNetworkAndWalletLoaded
+        ? storage.getPendingDepositsByHermezAddress(
+            pendingDeposits,
+            ethereumNetworkTask.data.chainId,
+            wallet.hermezEthereumAddress
+          )
+        : [],
+    [pendingDeposits, ethereumNetworkTask, wallet, ethereumNetworkAndWalletLoaded]
+  );
+  const accountPendingWithdraws = ethereumNetworkAndWalletLoaded
+    ? storage.getPendingWithdrawsByHermezAddress(
+        pendingWithdraws,
+        ethereumNetworkTask.data.chainId,
+        wallet.hermezEthereumAddress
+      )
+    : [];
+  const accountPendingDelayedWithdraws = ethereumNetworkAndWalletLoaded
+    ? storage.getPendingDelayedWithdrawsByHermezAddress(
+        pendingDelayedWithdraws,
+        ethereumNetworkTask.data.chainId,
+        wallet.hermezEthereumAddress
+      )
+    : [];
+  const accountTimerWithdraws = ethereumNetworkAndWalletLoaded
+    ? storage.getTimerWithdrawsByHermezAddress(
+        timerWithdraws,
+        ethereumNetworkTask.data.chainId,
+        wallet.hermezEthereumAddress
+      )
+    : [];
+  const pendingOnTopDeposits = accountPendingDeposits.filter(
+    (deposit) => deposit.type === TxType.Deposit
+  );
+  const pendingCreateAccountDeposits = accountPendingDeposits.filter(
+    (deposit) => deposit.type === TxType.CreateAccountDeposit
+  );
+  const coordinatorState = isAsyncTaskDataAvailable(coordinatorStateTask)
+    ? coordinatorStateTask.data
+    : undefined;
+  const fiatExchangeRates = isAsyncTaskDataAvailable(fiatExchangeRatesTask)
+    ? fiatExchangeRatesTask.data
+    : undefined;
+
+  React.useEffect(() => {
+    onChangeHeader();
+  }, [theme, onChangeHeader]);
+
+  React.useEffect(() => {
+    onCheckPendingDeposits();
+    onLoadPoolTransactions();
+    onLoadExits();
+    onCheckPendingWithdrawals();
+    onCheckPendingDelayedWithdrawals();
+  }, [
+    onCheckPendingDeposits,
+    onLoadPoolTransactions,
+    onLoadExits,
+    onCheckPendingWithdrawals,
+    onCheckPendingDelayedWithdrawals,
+  ]);
+
+  React.useEffect(() => {
+    const intervalId = setInterval(() => {
+      onCheckPendingDeposits();
+      onLoadPoolTransactions();
+      onLoadExits();
+      onCheckPendingWithdrawals();
+      onCheckPendingDelayedWithdrawals();
+    }, AUTO_REFRESH_RATE);
+
+    return () => {
+      clearInterval(intervalId);
+    };
+  }, [
+    onCheckPendingDeposits,
+    onLoadPoolTransactions,
+    onLoadExits,
+    onCheckPendingWithdrawals,
+    onCheckPendingDelayedWithdrawals,
+  ]);
+
+  React.useEffect(() => {
+    if (
+      pendingDepositsCheckTask.status === "successful" &&
+      poolTransactionsTask.status === "successful" &&
+      fiatExchangeRatesTask.status === "successful" &&
```

Comment:
I prefer not to use it inside of useEffect

Reply:
Agreed. You're right.

The reasoning is that we do not want asynchronous tasks to be launched when they are in the `reloading` state, to avoid re-fetching data which is already fetching.